docs/provisioner/08-01-provisioning-gcp.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Provision clusters on Google Cloud Platform (GCP)
+type: Tutorials
+---
+
+This tutorial shows how to provision clusters with Kyma Runtimes (Runtimes) on Google Cloud Platform (GCP).
+
+## Prerequisites
+
+- Existing project on GCP
+- Service account on GCP with the following roles:
+    * Compute Admin
+    * Kubernetes Engine Admin
+    * Kubernetes Engine Cluster Admin
+    * Service Account User
+- Key generated for your service account downloaded in the `json` format
+- Secret from the service account key created in the `compass-system` Namespace with the key `credentials` and the value encoded with base64
+
+## Provision Kyma Runtime on GCP
```

Comment:
We have a [template](https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/tutorials.md) for the `Tutorials` doc type that contains 3 amin but optional sections: `Prerequisites`, `Steps` & `Troubleshooting` - so I would make `Steps` out of this one with some actual steps if more than one is possible here...but still, we could adjust the heading I think